jax/experimental/global_device_array.py

```diff
@@ -372,6 +360,27 @@ def local_data(self, index) -> DeviceArray:
   def from_callback(cls, global_shape: Shape, global_mesh: pxla.Mesh,
                     mesh_axes: MeshAxes, data_callback: Callable[[Index],
                                                                  ArrayLike]):
+    """Constructs a GlobalDeviceArray via data fetched from ``data_callback``.
+
+    ``data_callback`` is used to fetch the data for each local slice of the returned GlobalDeviceArray.
+
+    Example::
```

Comment:
`Examples:`

Reply:
No, it doesn't become a code block if there is no double colon.